Oliver-ke/country-emergency
# #168501831 Setup react, webpack and babel

#### Description of Task to be completed?
- add react and reactdom
- add babel, webpack and scss loader
- configure react with webpack

 #### How should this be manually tested?
- clone this branch
- install dependencies using `npm install`
- run the dev script with `npm run dev`
- you should now see the app at `localhost:8080`

 #### Any background context you want to provide?
- ensure you have internet connection before installing dependencies

 #### What are the relevant pivotal tracker stories?
[#168501831](https://www.pivotaltracker.com/story/show/168501831)
 #### Screenshots (if appropriate)
 #### Questions:
